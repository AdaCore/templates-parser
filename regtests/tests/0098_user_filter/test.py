from test_support import *

gnatmake('user_filter')
run('user_filter')
