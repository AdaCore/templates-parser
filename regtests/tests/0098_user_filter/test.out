
tag_value

[F1=tag_value]

> [F1=v1]
> [F1=v2]
> [F1=v3]

> [F2=v1/azerty+tag_value]
> [F2=v2/azerty+tag_value]
> [F2=v3/azerty+tag_value]

tag_value .@SET. = //@SET\\
